src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,28 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
```

Comment:
log error with message

Reply:
I already have ERROR log in AuthenticationServiceImpl in login method, why i need 2 logs on the same error?

Reply:
the task was implement log in the TODO place

Reply:
So you just return in catch block without any action

Reply:
> the task was implement log in the TODO place

so, there is no TODO in the Main file